data/pokemon_upgrade_cost.json

```diff
@@ -0,0 +1,82 @@
+[
+[1, 200],
+[1, 200],
```

Comment:
How come this doesn't go [1,200],[2,200] and so forth? Sorry, there a whole lot of edits, still working through it all.